src/App.tsx

```diff
@@ -30,14 +33,14 @@ interface ApiMarkets {
     }[];
     products: string[];
     mapsLink: string;
+    name: string;
   }[];
-  zip: string;
 }
 
 export const App: React.FC = () => {
   const [allMarkets, setMarkets] = useState<ApiMarkets['markets']>([]);
   const [marketDetails, setDetails] = useState<ApiMarkets['marketDetails']>([]);
-  const [zip, setZip] = useState<ApiMarkets['zip']>('');
+  const [zip, setZip] = useState<string>('');
```

Comment:
Just curious why you changed this? I kind of liked it when all of the relevant props were in the interface up top for ease of viewing if you were new to the code base.

Reply:
it caused an issue when the interface is referenced on line 74. So the choice we have is to duplicate that big long interface without zip, or make zip declared as a string here. What's your preference?